pyproject.toml

```
@@ -110,6 +110,7 @@ dev = [
     # For testing Jupyter notebooks
```

Could you move this comment to be above `nbmake`?


```suggestion
    # pytest and its plugins
```